### DISCUSSION TITLE
Require as postcss plugin make webpack slow

### DISCUSSION THREAD

**Discussion by myamolane** [tailwindlabs/tailwindcss#4179] [Help]
After requiring tailwind as PostCSS plugin, it increases near 2 minutes for me.
I even didn't import any CSS file from tailwind.
I'm not very sure  if it's just because my project is kinda big since when I decrease my code, it works fine

Here are two pictures, first without tailwind and second with tailwind
![image](https://user-images.githubusercontent.com/26702359/115997120-81abaa80-a614-11eb-8778-49d81eaaa864.png)

![image](https://user-images.githubusercontent.com/26702359/115993940-92a1ef00-a607-11eb-898c-6b05d8ff8b96.png)

**Comment by adamwathan** [comment]
Hey! Yeah webpack gets really slow when you try to push any large CSS file through it, and Tailwind generates a 3mb CSS file by default.

We are tackling this problem with the new JIT mode which you can read more about here:

https://tailwindcss.com/docs/just-in-time-mode

**Reply by myamolane** [reply]
do tailwind with Postcss 7 support JIT
